seed/views/data_quality.py

```diff
@@ -92,12 +93,51 @@ def _get_severity_from_js(severity):
     return d.get(severity)
 
 
+class DataQualitySchema(AutoSchemaHelper):
+    def __init__(self):
+        super().__init__()
+
+        self.manual_fields = {
```

Comment:
I'm going to miss the magic that we had. But there is something nice to be said about being explicit.